progress/upload_status.py

```diff
@@ -66,12 +66,13 @@ def get_directory_status(directory, sample_sheet):
     # Must check status of upload to determine if upload is completed
     uploader_info_file = os.path.join(directory, STATUS_FILE_NAME)
     with open(uploader_info_file, "rb") as reader:
-        info_file = json.load(reader)
-        status = info_file[STATUS_FIELD]
-        if status in DIRECTORY_STATUS_LIST:
-            result.status = status
-        else:  # the status found in the file is not in the defined list
-            raise exceptions.DirectoryError("Cannot access directory", directory)
+        data = reader.read().decode()
```

Comment:
Reading from file straight into a json.load does not work in python 3.5 (load accepting strings vs bytes in different versions)
Reading and then decoding works on both 3.5 and 3.6+

Pulled functionality out of `with open(...)` since we are reading the full file into a dictionary and don't need it open.